Automattic/wp-calypso
# Hook Sites Dashboard into /sites routing

#### Proposed Changes

If the user clicks on the "All My Sites" link in the current site switcher while being on the site's dashboard, he will be redirected to the new Sites Dashboard.

The feature is kept behind the `build/sites-dashboard` feature flag.

#### Testing Instructions



1. Pull branch, start the calypso
2. Click on "Switch Sites" in the sidebar
3. Click on "All My Sites" in the sidebar
4. Confirm that the new Sites Dashboard is displayed instead of the Sites component
5. Choose any site, confirm that the home for the selected site is loaded

![Screen Capture on 2022-07-13 at 11-07-47](https://user-images.githubusercontent.com/727413/178696522-1bcd6d45-d6a2-44c7-9a28-3eb6e14da986.gif)



Related to https://github.com/Automattic/wp-calypso/issues/65181
